src/main/java/duke/command/AddDeadlineToList.java

```diff
@@ -0,0 +1,75 @@
+package duke.command;
+
+import duke.data.TaskList;
+import duke.exceptions.DukeException;
+import duke.exceptions.InvalidInputException;
+import duke.filemanager.Storage;
+import duke.task.Deadline;
+import duke.ui.Ui;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+import java.time.format.DateTimeFormatter;
+
+
+/**
+ * Handles the deadline command to add a new deadline to the list
+ */
+public class AddDeadlineToList extends Command {
+    private Deadline newTask;
+
+    /**
+     * preprocess userInput to create new deadline object
```

Comment:
Do check for capitalisation and uses of periods in comments. 


https://se-education.org/guides/conventions/java/basic.html#comments